packages/building-webpack/package.json

```
@@ -26,26 +31,34 @@
   ],
   "dependencies": {
     "@babel/core": "^7.3.3",
+    "@babel/polyfill": "^7.2.5",
     "@babel/preset-env": "^7.0.0",
-    "@babel/runtime-corejs2": "^7.2.0",
-    "@open-wc/polyfills-loader": "^0.2.2",
-    "@open-wc/webpack-import-meta-loader": "^0.1.3",
+    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
+    "@babel/plugin-syntax-import-meta": "^7.2.0",
+    "@webcomponents/webcomponentsjs": "^2.2.7",
     "babel-loader": "^8.0.0",
-    "babel-plugin-template-html-minifier": "^2.1.3",
-    "clean-webpack-plugin": "^1.0.0",
-    "fast-async": "^7.0.6",
+    "babel-plugin-template-html-minifier": "^2.2.1",
+    "babel-plugin-bundled-import-meta": "^0.3.0",
+    "clean-webpack-plugin": "^2.0.0",
+    "copy-webpack-plugin": "^5.0.1",
     "html-webpack-plugin": "^3.2.0",
+    "terser": "^3.17.0",
     "terser-webpack-plugin": "^1.2.1",
-    "webpack": "^4.28.0",
-    "webpack-babel-multi-target-plugin": "^v2.1.0-alpha.8"
+    "url-polyfill": "^1.1.5",
+    "webpack": "^4.28.0"
   },
   "devDependencies": {
+    "@babel/plugin-proposal-class-properties": "^7.3.4",
+    "@babel/preset-typescript": "^7.3.3",
     "@babel/register": "^7.0.0",
     "@webpack-cli/serve": "^0.1.3",
     "http-server": "^0.11.1",
     "lit-element": "^2.0.1",
+    "ts-loader": "^5.3.3",
+    "typescript": "^3.3.3333",
     "webpack-bundle-analyzer": "^3.0.3",
     "webpack-cli": "^3.2.1",
-    "webpack-dev-server": "^3.1.14"
+    "webpack-dev-server": "^3.1.14",
+    "webpack-merge": "^4.2.1"
```

```suggestion
    "webpack-merge": "^4.1.5"
```

we should depend on the same version in the monorepo